docs/releases/slashing-release/interfaces/slashing-parameter-configuration.md

```diff
@@ -192,75 +118,57 @@ interface IAVSDirectory {
+The function can be called with an EIP1271 signature from the operator or by the operator itself.
 
 Emits
 
-1. `MagnitudeUpdated` for each updated (operator, IStrategy, operatorSet)
-2. `TotalAndNonslashableMagnitudeUpdated` for each Strategy
+1. `MagnitudeAllocation` for each updated (operator, IStrategy, operatorSet)
 
 Reverts if
 
-1. The `allocatorSignature` is invalid or `msg.sender` is not the `operator`'s allocator
-2. if MagnitudeAdjustmentType == ALLOCATION:
-    - The magnitude queued for allocation to operatorSets is greater than the `operator`'s latest pending nonslashable magnitude for the `strategy` (cannot allocate more than nonslashable)
-3. if MagnitudeAdjustmentType == DEALLOCATION:
-    - The magnitude queued for deallocation from an operatorSet is greater than the `operator`'s latest pending magnitude for the `strategy` (cannot deallocate more than is allocated)
-4. There are > 3 pending magnitude updates for any of the operatorSets being updated for the given `operator` and strategy
-    1. This is due to gas constraints during slashing
-
-### queueMagnitudeDilution
+1. The `operatorSignature` is invalid or `msg.sender` is not the `operator`
+2. The magnitude queued for allocation to operatorSets summed with all pending allocations for the `(operator, strategy)` pair is greater less than the nonslashable magnitude for the `strategy` (cannot allocate more than nonslashable)
 
-Allocators call this and for each given strategy decreases all relative proportions of currently allocated magnitudes allowing the allocator to reduce the total slashable proportion. This is done cheaply by increasing their non slashable magnitude to the desired proportion of the new total magnitude. The function can be called with an EIP1271 signature from the operator's allocator or by the allocator itself.
+### queueDeallocation
 
-For example, to add 1% to an operator's nonslashable proportion for a given strategy, if the operator's total magnitude for the strategy was 100e18, their allocator would queue a dilution of 1.0101...e18 magnitude. This would result in their nonslashable magnitude being increased by 1% of the total magnitude.
+Operators call this to deallocate from their slashable stake (magnitudes) for a list of (operator, IStrategy, operatorSet(avs, operatorSetId)) tuples. Queued deallocations are no longer slashable after 21 days from the time of queueing.
+These deallocations must be completed in a 2-tx step process by calling `completeDeallocations` after 21 days have passed which increments the nonslashable magnitude of the operator, to be allocated to different operatorSets.
 
-Dilution reduces allocations from all existing operatorSets for the operator and strategy.
+The sum of all pending deallocations from a (operator, IStrategy, operatorSet(avs, operatorSetId)) tuple are verified to be less than the currently allocated magnitude. This is in order to make sure that the operator cannot deallocate more than they have allocated.
```

Comment:
Depending on storage don't think we can allow duplicate deallocations for the same strategy in this call. Probably should check for non duplicate strategy deallocations